### DISCUSSION TITLE
[2059790] 再刷一把 [本体]

### DISCUSSION THREAD

**Discussion by ffffsbbq** [pjy612/SteamManifestCache#411] [Share]
appid:
`appid:2059790`
keys:
```
2059791;682b0b49de2eb176cfc77be50e0cd9f60aa3f1ad39fbef2cfb16ad9f6b034490
```
Code:70CDE9E07B81DCA1
Email:fengshi2230@gmail.com
[2059571.zip](https://github.com/pjy612/SteamManifestCache/files/14000499/2059571.zip)

**Comment by pjy612** [comment]
参考 #26 
* 调整密钥格式

**Reply by ffffsbbq** [reply]
大佬 已修改

**Comment by ffffsbbq** [comment]
已修改

**Reply by pjy612** [reply]
* 密钥版式 错误 请参考 #26

**Reply by ffffsbbq** [reply]
改了

**Reply by ZEERDEER** [reply]
Key格式不对，请更改可直接复制的格式 #26

**Reply by ffffsbbq** [reply]
这下真对了
